lib/theme_check/checks/matching_schema_translations.rb

```diff
@@ -3,7 +3,7 @@ module ThemeCheck
   class MatchingSchemaTranslations < LiquidCheck
     severity :suggestion
     category :translation
-    doc docs_url("checks/docs/matching_schema_translations.rb")
+    doc docs_url("docs/checks/matching_schema_translations.md")
```

Comment:
The `doc` could default to `docs/checks/[NAME OF CHECK].md` (if the file exists), so you wouldn't even need to specify here.

Reply:
Yeah. My initial thoughts were that this was more explicit. So anybody that opens the project for the first time suddenly knows where to find the documentation files. Whereas by making it magic you need to learn the magic otherwise you wouldn't know.

But we could make it magic and make a unit test that fails when a doc file doesn't exist for a new check. Plus make a script that generates one for you.

Reply:
I'm adding all the docs and will change this after once every check is documented.